doc/source/overview/faq.md

```diff
@@ -0,0 +1,56 @@
+# Frequently Asked Questions
+
+## General troubleshooting
+
+**Q. I'm getting code errors using a method on my model and my data, what to do?**
+
+**A.** There can be many reasons why the method does not work. For code exceptions it is a good idea to check the following:
+ - Read the [docstrings](../api/modules.rst) of the method, paying close attention to the [type hints](https://docs.python.org/3/library/typing.html) as most errors are due to misformatted input arguments.
+ - Check that your model signature (it's type and expected inputs/outputs) are in the right format. For further details refer to [White-box and black-box models](../overview/white_box_black_box.md).
+
+**Q. The explanations are taking a long time to complete, what to do?**
+
+**A.** Explanations can take different times as a function of the model, the data and the explanation type itself. Refer to the [Introduction](../overview/high_level.md) and Methods sections for notes on algorithm complexity. You might need to experiment with the type of model, data points (specifically feature cardinality) and method parameters to ascertain if a specific method scales well for your use case.
+
+**Q. The explanation returned doesn't make sense to me, what to do?**
+
+**A.** Explanations reflect the decision-making process of the model and not that of a biased human observer, see [Biases](../overview/high_level.md#biases). Moreover, depending on the method, the data and the model, the explanations returned are valid but may be harder to interpret (e.g. see [Anchor explanations](#anchor-explanations) for some examples).
```

Comment:
the data and the model -> the data, and the model

Corrections suggested by Grammarly